src/libs/migrateOnyx.js

```diff
@@ -18,6 +20,8 @@ export default function () {
             RenamePriorityModeKey,
             AddEncryptedAuthToken,
             RenameExpensifyNewsStatus,
+            AddLastActionCreated,
```

Comment:
this `AddLastActionCreated` is really only here because it was improperly added in a previous PR.